File: Praktikum 5. Function/Soal 2/PRAK502-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
    return fabs(nilai1 - nilai2);
}

int mutlak(int angka)
{
    return (angka >= 0) ? angka : -angka;
}

int main()
{
    int a, b, c, d;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);

    int Hasil = hitung(a, c) + hitung(b, d);
    printf("%d", mutlak(Hasil));

    return 0;
}

File: Praktikum 5. Function/Soal 2/PRAK502-2310817310001-AvantioFierzaPatria.py
def hitung(nilai1, nilai2):
    return abs(nilai1 - nilai2)


def mutlak(angka):
    return angka if angka >= 0 else -angka


a = int(input())
b = int(input())
c = int(input())
d = int(input())

Hasil = hitung(a, c) + hitung(b, d)
print(mutlak(Hasil))
